x/concentrated-liquidity/position.go

```diff
@@ -704,14 +704,6 @@ func (k Keeper) transferPositions(ctx sdk.Context, positionIds []uint64, sender
 			return types.LockNotMatureError{PositionId: position.PositionId, LockId: lockId}
 		}
 
-		// Collect any outstanding incentives and rewards for the position.
-		if _, err := k.collectSpreadRewards(ctx, sender, positionId); err != nil {
```

Comment:
Let's update method spec to remove:
```
// It then collects any outstanding incentives and rewards for the position,
```

Reply:
Nice catch, updated in 8b0977fed54641554dcef683174d03d163fcccfe